bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/testable/TestArguments.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package io.ballerina.runtime.internal.testable;
+
+import io.ballerina.runtime.api.utils.StringUtils;
+import io.ballerina.runtime.api.values.BString;
+import io.ballerina.runtime.internal.scheduling.Strand;
+
+/**
+ * Parameters for testable main.
+ *
+ * @since 2201.6.0
+ */
+public class TestArguments {
+
+    private static final int ARGUMENTS_NUMBER = 10;
```

Comment:
Why did we take 10 as default value for args?

Reply:
This is the fixed number of arguments required for test execution (it's not including config args)